src/models/ban.ts

```
@@ -0,0 +1,32 @@
+  }
+}
+
+export async function getBanned(jid: string): Promise<boolean | undefined> {
+  if (!jid) return undefined;
+  const banned = await bannedUsers.findOne({ where: { jid } });
+  return banned !== undefined;
+}
+
+export async function removeBan(jid: string): Promise<boolean> {
+  const result = bannedUsers.destroy({ where: { jid } });
+  return result.changes ? true : false;
```

Biome (1.9.4)

[error] Unnecessary use of boolean literals in conditional expression.

Simplify your code by directly assigning the result without using a ternary operator.
If your goal is negation, you may use the logical NOT (!) or double NOT (!!) operator for clearer and concise code.
 Check for more details about NOT operator.
Unsafe fix: Remove the conditional expression with


(lint/complexity/noUselessTernary)